pkg/volume/cinder/attacher.go

```diff
@@ -124,7 +124,18 @@ func (attacher *cinderDiskAttacher) VolumesAreAttached(specs []*volume.Spec, nod
 		volumesAttachedCheck[spec] = true
 		volumeSpecMap[volumeSource.VolumeID] = spec
 	}
-	attachedResult, err := attacher.cinderProvider.DisksAreAttached(volumeIDList, string(nodeName))
+	instances, res := attacher.cinderProvider.Instances()
+	if !res {
+		return volumesAttachedCheck, fmt.Errorf("failed to list openstack instances")
```

Comment:
Indeed, this is copy/paste from the Attach method so I'll fix the comment there too.